lib/index.d.ts

```diff
@@ -4,6 +4,7 @@ import {
     Root as joi,
     ValidationOptions,
     ValidationError,
+    default as JoiDefault
```

Comment:
Is there any way we can reduce the number of imports from `joi` here? What is the difference between `joi` and `JoiDefault` here?

Reply:
Could indeed import Schema instead of JoiDefault, will modify the PR soon